lib/bootloader_setup.pm

```diff
@@ -403,6 +403,10 @@ sub specific_bootmenu_params {
         $args .= " addon=" . $addon_url;
     }
 
+    if (check_var('ISO_IN_EXTERNAL_DRIVE', 1)) {
```

Comment:
Are there other values than `1` to `ISO_IN_EXTERNAL_DRIVE` expected? Otherwise `get_var` seems to be a better choice me.